java-maven/maven-agent/pom.xml

```diff
@@ -47,6 +47,9 @@
                         <manifest>
                             <mainClass>com.rookout.HelloWorld</mainClass>
                         </manifest>
+                        <manifestEntries>
+                            <Rookout-Changlist>12345</Rookout-Changlist>
```

Comment:
This needs to be auto-deduced from Git/Perfoce.
Placing a constant isn't helpful to the customer :(

Reply:
I never worked with perforce - but i believe this is a good example: https://www.perforce.com/manuals/p4maven/index.html
Ill check GIT support as well